src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.util.List;
+
+
+/**
+ * Encapsulates a class extracted from the main logic to deal with the user interface through the command line
+ */
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String GREETING = "Hello from\n"  + LOGO + "\nHow can I help you ?\n" ;
+
+
+
+
+
+    /**
+     * Prints the ending message
+     * @return
+     */
+    public String goodbyeMessage() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+    /**
+     * Reads the User's input
+     * @return The String representation of the user's input
+     */
+
+
+    /**
+     * Prints the appropriate message for adding a Task
+     * @param task The task added
+     * @param size The current number of tasks
+     */
+    public String showAddCommand(Task task,int size) {
```

Comment:
Both of these methods are returning a String, but only the latter is a verb. Should you rename `goodbyeMessage` to a verb as well?